### PR TITLE
switch to relative urls for images and js files, fixed topImageHtml error

### PR DESCRIPTION
**Pull request by OmgImAlexis**
We now set the base URL and everything will be relative to that.

- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read [contribution guide](https://github.com/PyMedusa/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by OmgImAlexis** [comment]
Approved!

**Comment by fernandog** [comment]
Approved

**Comment by fernandog** [comment]
@OmgImAlexis fixing the conflicts?

File: gui/slick/views/displayShow.mako [hunk 1]
```diff
@@ -209,17 +209,17 @@
 
                 <table style="width:180px; float: right; vertical-align: middle; height: 100%;">
                     <% info_flag = subtitles.code_from_code(show.lang) if show.lang else '' %>
-                    <tr><td class="showLegend">Info Language:</td><td><img src="${srRoot}/images/subtitles/flags/${info_flag}.png" width="16" height="11" alt="${show.lang}" title="${show.lang}" onError="this.onerror=null;this.src='${srRoot}/images/flags/unknown.png';"/></td></tr>
+                    <tr><td class="showLegend">Info Language:</td><td><img src="images/subtitles/flags/${info_flag}.png" width="16" height="11" alt="${show.lang}" title="${show.lang}" onError="this.onerror=null;this.src='${srRoot}/images/flags/unknown.png';"/></td></tr>
```
Comment (ratoaq2) [inline gui/slick/views/displayShow.mako]: Should this
`this.src='${srRoot}/images/flags/unknown.png`
be changed to
`this.src='/images/flags/unknown.png`
?

Comment (OmgImAlexis) [inline gui/slick/views/displayShow.mako]: nope, the srRoot and the first `/` should be removed from all of these.

**Pull request merged**